scss/_patterns_grid.scss

```diff
@@ -179,4 +192,31 @@
       }
     }
   }
+
+  // Ways to break out of the 12-col grid
+
+  // Aside / main aligned to the left edge of the fixed-width grid
+  .row-new {
```

Comment:
Why is it called `-new`? Does it replace `.row`? Is it a variant?
It may be "new" now, but will not be in couple months.

Reply:
I actualy renamed it to row-breakout since  Ilast pushed - happy to change it to something else if you prefer

Reply:
Anything more semantic than 'new' would be better.

On the other hand, as I mentioned in my other comment rant (sorry for making it that long 😉 ) I'm starting to think that maybe trying to reuse the grid naming here is not that well, because it seems to be behaving different from the grid.